tests/run-programs.py

```diff
@@ -1,12 +1,42 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import subprocess
 import glob
 import time
 import sys
+from clvm_rs import deserialize_and_run_program, STRICT_MODE
+from clvm import KEYWORD_FROM_ATOM, KEYWORD_TO_ATOM
+from clvm.operators import OP_REWRITE
 
 ret = 0
 
+native_opcode_names_by_opcode = dict(
+    ("op_%s" % OP_REWRITE.get(k, k), op)
+    for op, k in KEYWORD_FROM_ATOM.items()
+    if k not in "qa."
+)
+
+for fn in glob.glob('programs/large-atom-*.hex'):
+
+    try:
+        program_data = bytes.fromhex(open(fn, 'r').read())
+        max_cost = 100000
+
+        cost, result = deserialize_and_run_program(
```

Comment:
Any chance we can isolate this ugly mess with the `native_opcode_names_by_opcode` and invocation of `deserialize_and_run_program` to one place in `tests`? I plan to refactor & simplify it soon anyway.

Reply:
well, I think it would make sense to bury it in the rust code, that would achieve the de-duplication